cmake/tribits/ctest_driver/TribitsCTestDriverCore.cmake

```diff
@@ -1004,31 +1004,46 @@ INCLUDE(${CMAKE_CURRENT_LIST_DIR}/TribitsCTestDriverCoreHelpers.cmake)
 # and other results and submitted and displayed on CDash (but not what CDash
 # site(s) or project(s) they are submitted to).  These options can all be set
 # in the CTest -S script using ``SET()`` statements before
-# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env.
+# ``TRIBITS_CTEST_DRIVER()`` is called and can be overridden in the env when
+# running the CTest -S driver script.
 #
 #   .. _CTEST_TEST_TYPE:
 #
 #   ``CTEST_TEST_TYPE=[Nightly|Continuous|Experimental]``
 #
-#     Determines the type of build.  This value is passed in as the first
+#     Determines the model for build.  This value is passed in as the first
```

Comment:
"the model for build" sounds strange to me.

Reply:
@jmgate, the term "model" is the correct term.  See:

* https://cmake.org/cmake/help/latest/command/ctest_start.html